tests/e2e/background_kyma_deletion_test.go

```diff
@@ -0,0 +1,76 @@
+package e2e_test
+
+import (
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
+)
+
+var _ = Describe("Background Kyma Deletion", Ordered, func() {
```

Comment:
I didn't do code review, but for the refactoring of e2e tests, I think for the existing e2e tests which delete kyma, it should use default `Background` deletion, and only have one dedicated for `Foreground`.